ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/aaa-authorization.j2

```diff
@@ -13,4 +13,10 @@ no aaa authorization config-commands
 {%     if aaa_authorization.commands.all_default is arista.avd.defined %}
 aaa authorization commands all default {{ aaa_authorization.commands.all_default }}
 {%     endif %}
+{%     if aaa_authorization.commands.privilege is arista.avd.defined %}
+{%         for command_levels in aaa_authorization.commands.privilege | arista.avd.natural_sort %}
+{%             set aaa_authorization_commands_privilege_cli = "aaa authorization commands " ~ command_levels.level ~ " default group " ~ command_levels.group %}
```

Comment:
Please change this var name to ```commands_privilege_cli```
As this var is used in aaa-authorization, it's ok to not mention ```aaa_authorization_``` at the beginning of var. Keep it simple and feel free to use full name of command (here it is commands privilege).

Reply:
Please change this var name to commands_privilege_cli